.eslintrc.json

```
@@ -0,0 +1,55 @@
+{
+  "root": true,
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "overrides": [
+    {
+      "files": ["**/*.tsx", "**/*.js"]
+    }
+  ],
+  "ignorePatterns": ["dist/**/*.js", "dist/**/*.d.ts"],
```

Nit: could just ignore the whole folder
```suggestion
  "ignorePatterns": ["dist/**/*"],
```